share/functions/help.fish

```diff
@@ -86,6 +86,14 @@ function help --description 'Show help for the fish shell'
                 set fish_browser $cmd[1]
             end
 
+            # Try and see if powershell.exe is an option to avoid UNC path error
+            # https://ss64.com/nt/cmd.html Launching CMD/batch files from a UNC path
+            if type -q powershell.exe
+                and set -l cmd (command -s powershell.exe cmd.exe /mnt/c/Windows/System32/cmd.exe)
```

Comment:
You misunderstood. I want the `powershell.exe` to go on line 84, inside the

```fish
if test -f /proc/version
   and string match -riq 'Microsoft|WSL|MSYS|MINGW' </proc/version
```

to ensure it only happens on WSL.